src/resdk/tables/base.py

```diff
@@ -213,6 +223,30 @@ def _metadata_version(self) -> str:
         )
         return version
 
+    @property
+    @lru_cache()
+    def _qc_version(self) -> str:
+        """Return server QC data version.
+
+        The versioning of QC Data on the server is determined by the hash of
+        the tuple of sorted QC data objects ids.
+
+        :return: data version
+        """
+        mqc_ids = []
+        for mqc in self.collection.data.filter(
+            type="data:multiqc", status="OK", fields=["id", "entity__id"]
+        ):
+            if mqc.sample is None:
+                continue
+            mqc_ids.append(mqc.id)
+        if len(mqc_ids) == 0:
+            raise ValueError(
+                f"Collection {self.collection.name} has no samples with MultiQC data!"
+            )
+
+        return str(hash(tuple(sorted(mqc_ids))))
```

Comment:
If you add ``ordering="id"`` to the filter when reading `mqs_ids` you can get rid of the `sorted` call.

Reply:
Good point, thanks.